apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,56 +1,41 @@
-import {
-	Button,
-	Header,
-	Loader,
-	RouterOutlet,
-	Sidebar,
-} from "~/libs/components/components.js";
-import { NAVIGATION_ITEMS } from "~/libs/constants/constants.js";
+import { RouterOutlet } from "~/libs/components/components.js";
 import { AppRoute } from "~/libs/enums/enums.js";
-import { useAppDispatch, useEffect, useLocation } from "~/libs/hooks/hooks.js";
+import {
+	useAppDispatch,
+	useAppSelector,
+	useEffect,
+	useLocation,
+} from "~/libs/hooks/hooks.js";
+import { actions as authActions } from "~/modules/auth/auth.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
-import { mockImages } from "../carousel/assets/mock-images/mock-images.js";
-import { Carousel } from "../carousel/carousel.js";
 import styles from "./styles.module.css";
 
 const App = (): React.JSX.Element => {
 	const { pathname } = useLocation();
 	const dispatch = useAppDispatch();
-	const mockUserWithoutAvatar = {
-		avatarUrl: null,
-		firstName: "John",
-		lastName: "Smith",
-	};
 
 	const isRoot = pathname === AppRoute.APP;
 
+	const hasToken = useAppSelector(({ auth }) => auth.authenticatedUser);
+
 	useEffect(() => {
 		if (isRoot) {
 			void dispatch(userActions.loadAll());
 		}
 	}, [isRoot, dispatch]);
 
+	useEffect(() => {
+		if (!hasToken) {
+			void dispatch(authActions.getAuthenticatedUser());
+		}
+	}, [dispatch, hasToken]);
```

Comment:
The logic should be the opposite - if token exists - fetch auth user

```suggestion
	useEffect(() => {
		if (hasToken) {
			void dispatch(authActions.getAuthenticatedUser());
		}
	}, [dispatch, hasToken]);
```